system/maintenance-controller/templates/manager.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: maintenance-controller-config
+  namespace: kube-system
+type: Opaque
+stringData:
+{{ tpl (.Files.Glob "maintenance.yaml").AsConfig . | indent 2 }}
```

Comment:
As an alternative to the included config template it would be nice of the chart allows to specify the complete content of the `maintenance.yaml` as a value. This would allow reusing this chart with a config that is completely different (e.g. no flatcar OS involved)